bcp47/library/Data/BCP47/Internal/Extension.hs

```diff
@@ -27,15 +29,18 @@ import Text.Megaparsec.Error (errorBundlePretty)
 -- is commonly used in association with languages or language tags but
 -- that is not part of language identification.
 --
-newtype Extension = Extension { extensionToText :: Text }
+newtype Extension = Extension { unExtension :: CIText }
   deriving stock (Show, Eq, Ord)
 
+extensionToText :: Extension -> Text
+extensionToText = CI.original . unExtension
```

Comment:
:ok_hand: I always forget exporting the selector enables updates.

Reply:
Yeeep. To me, this is what _does_ make this a change in functionality since you can no longer do updates (but again, `Internal` module, so no guarantees)

Reply:
Yeah, I'm fine with that nuanced change in a `.Internal`.